public/app/plugins/panel/geomap/editor/MapViewEditor.tsx

```diff
@@ -112,6 +112,37 @@ export const MapViewEditor: FC<StandardEditorProps<MapViewConfig, any, GeomapPan
         </InlineField>
       </InlineFieldRow>
 
+      {value?.id === MapCenterID.Fit && (
+        <>
+          <InlineFieldRow>
+            <InlineField label="Padding %" labelWidth={labelWidth} grow={true}>
+              <NumberInput
+                value={value?.padding ?? 5}
+                min={0}
+                step={1}
+                onChange={(v) => {
+                  onChange({ ...value, padding: v });
+                }}
+              />
+            </InlineField>
+          </InlineFieldRow>
+          <InlineFieldRow>
+            <InlineField label="Data Scope" labelWidth={labelWidth} grow={true}>
+              <RadioButtonGroup
+                value={value?.lastOnly ? 'last' : 'all'}
+                options={[
+                  { label: 'All', value: 'all' },
+                  { label: 'Last Only', value: 'last' },
```

Comment:
For values it might make sense to create an enum type for better maintainability

Reply:
Agreed.